fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -1378,6 +1378,13 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueueData* self, RelocateData rd,
 			} else {
 				TEST(true); // move to removed server
 				healthyDestinations.addDataInFlightToTeam(-metrics.bytes);
+
+				auto inFlightRange2 = self->inFlight.rangeContaining(rd.keys.begin);
+				ASSERT(inFlightRange2.range() == rd.keys);
+				ASSERT(inFlightRange2.value().randomId == rd.randomId);
+				inFlightRange2.value().cancellable = false;
```

Comment:
Oh whoops i meant to set it back to cancellable=true here since it isn't moving.
This isn't strictly required though, the fix is on the line below clearing completeDests, so i'll just remove it